SpatialGDK/Source/SpatialGDK/Public/Interop/Connection/ConnectionConfig.h

```diff
@@ -223,10 +223,6 @@ class FReceptionistConfig : public FConnectionConfig
 		if (!Host.IsEmpty())
 		{
 			ReceptionistHost = Host;
-			if (ReceptionistHost.Compare(SpatialConstants::LOCAL_HOST) != 0)
```

Comment:
Should `UseExternalIp = false` be added to `LoadDefaults` ?

Reply:
Actually that looks like it will potentially break the `SpatialConstants::RECONNECT_USING_COMMANDLINE_ARGUMENTS` flow...

Reply:
Good call

Reply:
> Actually that looks like it will potentially break the `SpatialConstants::RECONNECT_USING_COMMANDLINE_ARGUMENTS` flow...

I think it's fine. `LoadDefaults` will only be called if parsing the command line failed. And in that case it should be reasonable to assume we want `UseExternalIp = false`